torchinductor/utils.py

```diff
@@ -94,12 +94,14 @@ def gen_gm_and_inputs(target, args, kwargs):
 
 
 def timed(model, example_inputs, times=1):
-    synchronize()
+    if torch.cuda.is_available():
+        synchronize()
```

Comment:
it would be a tiny tad cleaner to just have a `def synchronize()` that does the `torch.cuda.is_available()` and `torch.cuda. synchronize()` inside it. Less repetition, and fits nicely into this utils.py class